### PR TITLE
Update copyright

### PR DESCRIPTION
**Pull request by kdheepak**

### CODE REVIEW

**Comment by phlptp** [comment]
There are a few edge cases where the code was modified vs co-developed that we may have to update later

File: examples/CInterface/nonlings_fed2.c [hunk 1]
```diff
@@ -1,10 +1,6 @@
 /*
-Copyright (C) 2017-2018, Battelle Memorial Institute
-All rights reserved.
-
-This software was co-developed by Pacific Northwest National Laboratory, operated by the Battelle Memorial
-Institute; the National Renewable Energy Laboratory, operated by the Alliance for Sustainable Energy, LLC; and the
-Lawrence Livermore National Laboratory, operated by Lawrence Livermore National Security, LLC.
+Copyright © 2017-2018, Battelle Memorial Institute; Lawrence Livermore National Security, LLC; Alliance for Sustainable Energy, LLC
```
Comment (phlptp) [inline examples/CInterface/nonlings_fed2.c]: This line is longer than would be allowed by clang format so will get changed when I next run that over the files

is it possible to divide this into 2 lines in a nice fashion
maybe
```
Copyright © 2017-2018, 
Battelle Memorial Institute; Lawrence Livermore National Security, LLC; Alliance for Sustainable Energy, LLC
```

or 

```
Copyright © 2017-2018, Battelle Memorial Institute; Lawrence Livermore National Security, LLC; 
Alliance for Sustainable Energy, LLC
```

Comment (kdheepak) [reply examples/CInterface/nonlings_fed2.c]: I'll divide these lines in the first manner.

**Comment by kdheepak** [comment]
I don't think the copyright symbol is going to cause issues. They have been building fine on my machine / on travis.

**Comment by phlptp** [comment]
I know you had some other issues before where you took out non-standard symbols so that is why I was asking.

**Comment by kdheepak** [comment]
I ran the same test that I had before that caused issues last time, but this didn't throw any issues. Previously, when I opened it in vim I'd see weird characters e.g. \ue91 etc, and those were the ones I had to change. Now it shows the proper copyright symbol, so I think we are good.

**Comment by phlptp** [comment]
ok, if you update the line splits, I think we can call it good

**Pull request merged**